eventkit_cloud/tasks/task_factory.py

```diff
@@ -187,7 +187,6 @@ def create_run(job_uid, user=None):
         # https://docs.djangoproject.com/en/1.10/topics/db/transactions/#django.db.transaction.atomic
         # enforce max runs
         with transaction.atomic():
-            max_runs = settings.EXPORT_MAX_RUNS
```

Comment:
The `EXPORT_MAX_RUNS` setting is designed to let jobs have multiple runs.  We don't necessarily support this right now in the UI, but I'm not sure it makes sense to remove it at this point.  Even if we did we would have to remove other references to it, otherwise it could be confusing.  It may make more sense to keep that logic in here for now, unless there is a fundamental issue with keeping possibly many runs and fixing this bug.   You could probably keep the old code with something like: `jobs.runs.filter(deleted=False).earliest(field_name='started_at').soft_delete(user=user)`